### ISSUE TITLE
Inline option not working as expected

### ISSUE THREAD

**Issue by tjbenton** [sasstools/sass-lint#465]
I'm getting a parser error in the following code so I added the scss-lint inline option to disable the linting for the specific section that's getting a parser error. That way the rest of the file could get parsed correctly and I can see other potential errors in the rest of the file. However the inline disable/enable option isn't preventing the parser error as expected, instead it's still returning an error. While this isn't very common to do it shouldn't return an error because it's the only way you can reference dynamic properties. 

Is there something I'm doing wrong, or something I need to do to prevent this error?

```scss
@mixin placeholder-animation($arg) {
  // scss-lint: disable-all
  @each $property, $value in parse-animation($arg) {
    #{$property}: $value;
  }
  // scss-lint: enable-all
  $name: get-animation-type($arg, 'name');
  @if index($animations, $name) {
    @keyframes #{$name} {
      @content;
    }
  }
}
```

![screen shot 2015-12-23 at 8 57 39 am](https://cloud.githubusercontent.com/assets/4776422/11977702/472e930e-a953-11e5-8d0a-e97a902576e6.jpg)

#70, #402, and #462

**Comment by DanPurdy** [comment]
Firstly, this is `sass-lint` not [scss-lint](https://github.com/brigade/scss-lint) so of course scss-lint disable wont work! :smile: 

Secondly we currently don't have the option to disable rules across lines as I explained to you in the last issue you raised, it is being worked on though.

Thirdly even if the option was there it wouldn't remove that parse error as you expect it should, parsing of `Sass` / `scss` happens before any rules are applied and disabling rules doesn't stop the parser from having to read the entire file and encountering problems. The parser is gonzales-pe and it currently doesn't support interpolation like you're using there, see [here](https://github.com/tonyganch/gonzales-pe/issues/116) this is something we don't have any control over unfortunately.

Sorry but closing this as a duplicate of a few of our other issues with regards to interpolation parse errors.
